mailet/api/pom.xml

```diff
@@ -55,11 +55,16 @@
             <version>1.7.5</version>
```

Comment:
You miss tests on this commit:
 - invalidation of parsing upon:
    - setMessage...
    - getMessage().addHeader() ...